packages/ui/src/composites/wui-search-bar/index.ts

```diff
@@ -0,0 +1,35 @@
+import { html, LitElement } from 'lit'
+import { customElement } from 'lit/decorators.js'
+import { resetStyles } from '../../utils/ThemeUtil'
+import '../../composites/wui-input-element'
+import '../../composites/wui-input'
+import { searchSvg } from '../../assets/svg/search'
+import { closeSvg } from '../../assets/svg/close'
+import styles from './styles'
+
+@customElement('wui-search-bar')
+export class WuiSearchBar extends LitElement {
+  public static styles = [resetStyles, styles]
+
+  // -- render ------------------------------------------------------- //
+  public render() {
+    return html`<wui-input placeholder="Search wallet" .icon=${searchSvg} size="sm"
+      ><wui-input-element @click=${this.clearValue} .icon=${closeSvg}></wui-input-element
+    ></wui-input>`
+  }
+
+  private clearValue() {
+    const wuiInput = this.shadowRoot?.querySelector('wui-input')
```

Comment:
I think more correct way of doing this is to reference input element via `ref` 
https://lit.dev/docs/templates/directives/#referencing-rendered-dom